setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+
+with open('README.md', 'r') as f:
+    long_description = f.read()
+
+
+setuptools.setup(
+    name='arcus-python',
```

Comment:
this should be `arcus` to be consistent with https://pypi.org/project/arcus

Reply:
done